### PR TITLE
[I18N] l10n_es: add catalan translation

### PR DESCRIPTION
**Pull request by santiagopim**
* addons/l10n_es/i18n/ca.po: Add the catalan language translation for the spanish localization.
* addons/l10n_es/data/template/account.account-es_assec.csv: Add name@ca field with account names in catalan.
* addons/l10n_es/data/template/account.account-es_common.csv: Add name@ca field with account names in catalan.
* addons/l10n_es/data/template/account.account-es_full.csv: Add name@ca field with account names in catalan.
* addons/l10n_es/data/template/account.account-es_pymes.csv: Add name@ca field with account names in catalan.
* addons/l10n_es/data/template/account.group-es_common.csv: Add name@ca field with account names in catalan.

Description of the issue/feature this PR addresses: Add the name@ca for catalan translation at the main account plans in the spain l10n_es localization, and the l10n_es/i18n/ca.po file too.

Current behavior before PR: The accounts names are in english when user configures her environment to use the catalan language.

Desired behavior after PR is merged: The accounts names are in catalan.

Previously crashed PR https://github.com/odoo/odoo/pull/160486

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by jco-odoo** [comment]
@santiagopim Please let me know if you can adapt for the cooperativas as well now that https://github.com/odoo/odoo/pull/159357 has been merged.  Moltes gràcies!

**Comment by santiagopim** [comment]
Ok, working on it. Couple days ...

File: addons/l10n_es/data/template/account.fiscal.position-es_common.csv
Comment (santiagopim) [inline addons/l10n_es/data/template/account.fiscal.position-es_common.csv:1]: Mmm, blank values stored as ,, originally were ,"",. Must redo CSV export.

File: addons/l10n_es/data/template/account.tax-es_common.csv
Comment (santiagopim) [inline addons/l10n_es/data/template/account.tax-es_common.csv:1]: Mmm, blank values stored as ,, originally were ,"",. Must redo CSV export.

**Comment by santiagopim** [comment]
@jco-odoo: The commit is adapted to 'cooperativas' now. Hope tests will pass. Thanks, i de res.

**Comment by jco-odoo** [comment]
@robodoo r+

File: addons/l10n_es/i18n/ca.po [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+#, python-format
+msgid "Common"
+msgstr "Comú"
+
+#. module: l10n_es
+#. odoo-python
+#: code:addons/l10n_es/models/template_es_full.py:0
+#, python-format
+msgid "Complete (2008)"
+msgstr "Complert (2008)"
```
[the remark this answers is not in the document]
Comment (santiagopim) [reply addons/l10n_es/i18n/ca.po:36]: Oh, yes. Should be 'Complet (2008)'.

**Comment by santiagopim** [comment]
@jco-odoo Ok, this PR has been merged into 17.0, but the forward-port to master has failed with conflicts in one file. So saas-17.1, saas-17.2, and master, none of them include this code. What must I do ?

Thanks

**Comment by santiagopim** [comment]
@jbw-odoo Hello, text is fine, but differing line endings. Don't know if this can cause problems.

- yours: DOS/Windows (CR-LF)
- 17.0: UNIX (LF)

Thanks

**Pull request closed without merging**